src/tab-list.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class TabList extends React.Component {
+  constructor() {
+    super();
+    this.state = { focus: false };
+    this.handleBlur = this.handleBlur.bind(this);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+    this.handlers = {
+      ArrowLeft: this.previous.bind(this),
+      ArrowRight: this.next.bind(this),
+    };
+  }
+
+  wrapIndex(index) {
+    const count = React.Children.count(this.props.children);
+    return (index + count) % count;
+  }
+
+  handleBlur() {
+    this.setState({ focus: false });
+  }
```

Comment:
Hmmm, wondering if we can avoid needing this state by:

- Creating a React ref for each of the tabs and passing it down as a prop
- Focusing the ref instance from here instead of changing the `focus` state and passing it down